packages/syft/src/syft/core/node/common/node_manager/redis_store.py

```diff
@@ -18,13 +19,15 @@
 from ....common.uid import UID
 from ....node.common.node_table.bin_obj_dataset import BinObjDataset
 from ....store import ObjectStore
+from ....store.proxy_dataset import ProxyDataClass
 from ....store.storeable_object import StorableObject
 from ..node_table.bin_obj_metadata import ObjectMetadata
 
 
 class RedisStore(ObjectStore):
-    def __init__(self, db: Session) -> None:
+    def __init__(self, db: Session, settings: BaseSettings) -> None:
         self.db = db
+        self.settings = settings
```

Comment:
Q: Why do we pass settings to Object Store, I think we are not using it now, or will it be used later?

Reply:
@rasswanth-s we will be using this later, there is some work still left on refactoring this part of the code.